### PR TITLE
It was added the stochastic part of the model

### PR DESCRIPTION
**Pull request by LuanDaSilva**
This function allows the use of vector as a parameter of function/object. It reads the vector(s) and generate a proper input to the function/object returning a list of function/objects outputs evalueted at each vector(s) value.

### CODE REVIEW

File: ross/utils.py [hunk 1]
```diff
@@ -4,6 +4,23 @@
 from bokeh.models import LogColorMapper
 from bokeh.palettes import Viridis256
 from bokeh.plotting import figure
+import itertools
+from collections import Iterable
+
+from ross.materials import Material
```
Comment (raphaeltimbo) [inline ross/utils.py]: Is this import being used?

File: ross/utils.py [hunk 2]
```diff
@@ -4,6 +4,23 @@
...
+
+def stochastic_ross(f,*args):
```
Comment (raphaeltimbo) [inline ross/utils.py]: Please add the function documentation. Also check if we could improve the name of this function to give a better idea of its purpose to the user.

**Comment by rodrigomoliveira1** [comment]
I don't why I getting an error on `test_fluid_flow.py` in this PR. Once again we're experiencing strange behaviors from the CI's. I'm trying small commits to see if it vanishes eventually like the last time.

**Comment by rodrigomoliveira1** [comment]
PR #478 moves the method once added in utils.py to new folder.
So, I'm closing this PR.

**Pull request closed without merging**